eng/Version.Details.xml

```diff
@@ -64,6 +64,10 @@
       <Uri>https://github.com/aspnet/AspNetCore</Uri>
       <Sha>1937c804fa83ee3adddc7e648c610f880c24188e</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.3.1" Version="1.0.2-beta4.19465.1" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
```

Comment:
```suggestion
    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.3.1" Version="1.0.2-beta4.19465.1">
```

test-templates don't flow into toolset, so we do not need any coherent parent with it here. (I think I was confused over your shoulder earlier, sorry.)

Reply:
Done